tests/langservice/detectors/test_detectors_leakreplay.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+from garak.attempt import Attempt
+from garak.detectors.leakreplay import StartsWith
+
+
+def reverse_translate(attempt) -> Attempt:
+    from garak.langservice import _load_translator
+
+    translator_config = {
+        "language": "ja-en",
+        "model_type": "local",
+    }
+    reverse_translator = _load_translator(translator_config)
+    for i, thread in enumerate(attempt.messages):
+        for j, turn in enumerate(thread):
+            if turn["role"] == "assistant":
+                attempt.messages[i][j]["content"] = "\n".join(
+                    reverse_translator.translate_prompts(turn["content"].split("\n"))
+                )
+    return attempt
+
+
+@pytest.mark.requires_storage(required_space_gb=2, path="/")
+def test_startswith_detect():
+    detector = StartsWith()
+
+    attempt = Attempt(prompt="This is a test prompt")
+    attempt.notes["triggers"] = ["Hello", "Test"]
+    attempt.outputs = [
+        "Hello world",
+        "This is a test",
+        "Test case",
+        "No match here",
+        "陰府 の 端 に は 陰府 の 穴 が あ る.",
```

Comment:
The pattern we've often used elsewhere is to make values like "Hello" and "Test" constants and then refer to those in both `outputs` and e.g. `triggers`